gui/wxpython/datacatalog/tree.py

```diff
@@ -284,7 +235,7 @@ def label(self):
 
         return _("{name}").format(**data)
 
-    def match(self, **kwargs):
+    def match(self, exact=True, **kwargs):
         """Method used for searching according to given parameters.
 
         :param value: dictionary value to be matched
```

Comment:
key, value are really part of kwargs parameter, so would be more clear to document it as what kwargs is. Allowed values for kwargs would be nice, at least what they can be in general.

exact=True seems quite straightforward to understand. However, implementation exact=False is surprising as name is expected to be compiled regular expression and also no other keys than name and type are considered/implemented. Probably the best/sufficient implementation for what we need, but documenting this explicitly would be quite helpful since there is only little consistency/correspondence between the two behaviors.